src/components/UI/ClusterCreation/InstanceTypeSelection.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from '@emotion/styled';
+
+import { AdditionalInputHint } from './StyledInput';
+
+export const InstanceTypeDescription = styled(AdditionalInputHint)`
+  margin: 0 0 0
+    ${({ theme }) =>
+      /* eslint-disable-next-line no-magic-numbers */
+      theme.spacingPx * 4}px;
```

Comment:
Not a big fan of this notation, I'd prefer to have all margin sizes on the same line, but not a blocker

Reply:
yeah..

> Looks like the magic number linter rule is making our code really dirty. Is it maybe time to disable it?

Fine for me. We could ignore 1-9 in the rule config for starters.